hashicorp/web-unified-docs
# chore: Remove Vault content

# Changes

- Remove vault content from code base to prep for education engineer test. This includes:
  - Removing vault from the `content` dir
  - Removing vault from the `public/assets` dir
- Commenting out Vault in repo config

- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881343/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/vault/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1148" alt="Screenshot 2024-10-16 at 16 18 58" src="https://github.com/user-attachments/assets/df7a997b-009a-4a05-9ea8-1e5c603e293b">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-f79f4b-hashicorp.vercel.app/api/content/vault/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
